### PR TITLE
Hero and User Dtos and facades interfaces added

### PR DESCRIPTION
**Pull request by MiroslavLuckai**

### CODE REVIEW

File: msa-api/src/main/java/cz/muni/fi/pa165/msa/facade/UserFacade.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package cz.muni.fi.pa165.msa.facade;
+
+import cz.muni.fi.pa165.msa.dto.UserDTO;
+
+import java.util.List;
+
+public interface UserFacade {
+
+    Long createUser(UserDTO user);
```
Comment (MichaelaBajanova) [inline msa-api/src/main/java/cz/muni/fi/pa165/msa/facade/UserFacade.java:9]: missing java doc

File: msa-api/src/main/java/cz/muni/fi/pa165/msa/facade/HeroFacade.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
+package cz.muni.fi.pa165.msa.facade;
+
+import cz.muni.fi.pa165.msa.dto.HeroDTO;
+
+public interface HeroFacade {
+
+    Long createHero(HeroDTO hero);
```
Comment (MichaelaBajanova) [inline msa-api/src/main/java/cz/muni/fi/pa165/msa/facade/HeroFacade.java:7]: missing java doc

File: msa-api/src/main/java/cz/muni/fi/pa165/msa/dto/HeroDTO.java [hunk 3]
```diff
@@ -0,0 +1,75 @@
+package cz.muni.fi.pa165.msa.dto;
+
+import cz.muni.fi.pa165.monsterslayeragency.enums.Skill;
+
+import java.util.Objects;
+import java.util.Set;
+
+public class HeroDTO {
+
```
Comment (MichaelaBajanova) [inline msa-api/src/main/java/cz/muni/fi/pa165/msa/dto/HeroDTO.java:9]: why no id attribute?

Comment (MiroslavLuckai) [reply msa-api/src/main/java/cz/muni/fi/pa165/msa/dto/HeroDTO.java:9]: Id attribute added.

**Pull request merged**